usr.sbin/bluetooth/bcmfw/bcmfw.8

```diff
@@ -29,7 +29,7 @@
 .Os
 .Sh NAME
 .Nm bcmfw
-.Nd firmware download utility for Broadcom BCM2033 chip based Bluetooth USB devices
+.Nd install firmware for Broadcom BCM2033 Bluetooth USB devices
```

Comment:
Install isn't quite the right word here...  These utilities download the firmware to the device. This usually isn't called 'installing' so I think this part would make things more confusing.